src/pages/transactions/create.js

```diff
@@ -0,0 +1,20 @@
+import { useRouter } from "next/router";
+import Form from "@/src/components/Form";
+
+export default function Create({ onAddTransaction }) {
+  const router = useRouter();
+
+  function handleAdd(data) {
+    onAddTransaction(data);
+    router.push("/");
+  }
+
+  return (
+    <Form
+      onSubmitForm={handleAdd}
+      formTitle={"Add Transaction"}
+      confirmButtonText={"Add"}
+      add
```

Comment:
I guess this is the mode to handle the form based on the fact if it's in "edit" or "add" mode?

Reply:
yes, changed the values to "addMode" and "editMode"